inc/namespace.php

```diff
@@ -128,6 +128,7 @@ function add_login_logo() {
  */
 function load_plugins() {
 	require_once ROOT_DIR . '/vendor/stuttter/wp-user-signups/wp-user-signups.php';
+	require_once ROOT_DIR . '/vendor/johnbillion/extended-cpts/extended-cpts.php';
```

Comment:
I had a hunch and checked the `extended-cpts` composer file and it already autoloads this file: https://github.com/johnbillion/extended-cpts/blob/develop/composer.json#L57

The addon has no side effects so it's fine for it to be loaded that way - basically you can just remove this line!

Reply:
Does `extended-cpts.php` have side effects? If so, I think that might be the only package we have that would automaticlly execute side effects.

Reply:
@joehoyle there are actually no side effects from `extended-cpts` so I think we'll be good removing that line as @roborourke said. 😄